src/main/java/de/mediathekview/mserver/crawler/br/tasks/BrWebAccessHelper.java

```diff
@@ -1,41 +1,41 @@
-/*
- * BrWebAccessHelper.java
- * 
- * Projekt    : MServer
- * erstellt am: 07.10.2017
- * Autor      : Sascha
- * 
- * (c) 2017 by Sascha Wiegandt
- */
-package de.mediathekview.mserver.crawler.br.tasks;
-
-import java.net.MalformedURLException;
-
-import org.apache.logging.log4j.Logger;
-
-import com.google.gson.JsonSyntaxException;
-
-import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
-import de.mediathekview.mserver.crawler.br.BrWebAccessExecution;
-
-public class BrWebAccessHelper<V> {
-
-    public static void handleWebAccessExecution(Logger log, AbstractCrawler crawler, BrWebAccessExecution action) {
-        try {
-            action.run();
-        } catch (final JsonSyntaxException jsonSyntaxException) {
-            handleError("The json syntax for the BR task to get all Sendungen has an error.", jsonSyntaxException, log, crawler);
-        } catch (final MalformedURLException malformedURLException) {
-            handleError("The URL given to WebAccess is not valid.", malformedURLException, log, crawler);
-        } catch (final IllegalArgumentException illegalArgumentException) {
-            handleError("URL given to the WebAccess was Null", illegalArgumentException, log, crawler);
-        }
-    }
-    
-    private static void handleError(String errorMessage, Exception e, Logger log, AbstractCrawler crawler) {
-        log.error(errorMessage, e);
-        crawler.incrementAndGetErrorCount();
-        crawler.printErrorMessage();
-    }
-    
-}
+/*
+ * BrWebAccessHelper.java
+ * 
+ * Projekt    : MServer
+ * erstellt am: 07.10.2017
+ * Autor      : Sascha
+ * 
+ * (c) 2017 by Sascha Wiegandt
+ */
+package de.mediathekview.mserver.crawler.br.tasks;
+
+import java.net.MalformedURLException;
+
+import org.apache.logging.log4j.Logger;
+
+import com.google.gson.JsonSyntaxException;
+
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.br.BrWebAccessExecution;
+
+public class BrWebAccessHelper<V> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') V is not used in the class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)